product/models.py

```diff
@@ -70,7 +70,8 @@ class Meta:
 
 class Image(models.Model):
     img_url           = models.CharField(max_length=300)
-    middle_table      = models.ForeignKey("MiddleColorImage", on_delete=models.SET_NULL, null =True)
+    product           = models.ForeignKey("Product", on_delete=models.SET_NULL, null =True)
+    sub_img           = models.BooleanField(default=False)
```

Comment:
sub img 서브 이미지인지 아닌지 확인하는 필드인가요 ? 그렇다면 단에 앞에 is를 붙여서 명확하게 해주세요~!
```suggestion
    is_sub_img           = models.BooleanField(default=False)
```